src/natcap/invest/urban_nature_access.py

```diff
@@ -0,0 +1,322 @@
+import logging
+import os
+import shutil
+import tempfile
+
+import numpy
+import pygeoprocessing
+import taskgraph
+from osgeo import gdal
+from osgeo import osr
+
+from natcap.invest import MODEL_METADATA
+from . import spec_utils
+from . import utils
+from . import validation
+from .spec_utils import u
+
+LOGGER = logging.getLogger(__name__)
+UINT32_NODATA = int(numpy.iinfo(numpy.uint32).max)
+FLOAT32_NODATA = float(numpy.finfo(numpy.float32).max)
+ARGS_SPEC = {
+    'model_name': MODEL_METADATA['urban_nature_access'].model_title,
+    'pyname': MODEL_METADATA['urban_nature_access'].pyname,
+    'userguide_html': MODEL_METADATA['urban_nature_access'].userguide,
+    'args_with_spatial_overlap': {
+        'spatial_keys': [
+            'lulc_raster_path', 'population_raster_path',
+            'admin_unit_vector_path'],
+        'different_projections_ok': True,
+    },
+    'args': {
+        'workspace_dir': spec_utils.WORKSPACE,
+        'results_suffix': spec_utils.SUFFIX,
+        'n_workers': spec_utils.N_WORKERS,
+        'lulc_raster_path': {
+            **spec_utils.LULC,
+            'projected': True,
+            'projection_units': u.meter,
+            'about': (
+                "A map of LULC codes. "
+                "All values in this raster must have corresponding entries "
+                "in the LULC attribute table.  This raster must be linearly "
+                "projected in meters."),
```

Comment:
Sorry about the run around here, but I just remembered that with Emily's work with `pint` we were dropping these redundant descriptions since they should be represented from the `projected` and `projected_units` attributes.